kotishell/include/tokenize.h

```diff
@@ -24,33 +24,44 @@
 
 typedef enum e_tk_token_type
 {
-	TK_WORD,
-	TK_PIPE,
-	TK_HEREDOC,
-	TK_NEWLINE,
-	TK_LESS,
-	TK_DGREAT,
+	TK_DOLLAR,
 	TK_GREAT,
 	TK_EMPTY,
-	TK_WHITESPACE,
 	TK_EOL,
+	TK_LESS,
+	TK_NEWLINE,
+	TK_OTHER,
+	TK_PIPE,
+	TK_QUOTE_1,
+	TK_QUOTE_2,
+	TK_QUESTION,
+	TK_WHITESPACE,
+	TK_WORD,
+	TK_BUG,
 }	t_tk_token_type;
 
 typedef enum e_tk_state
 {
-	TK_ST_WHITESPACE,
-	TK_ST_WORD,
-	TK_ST_START,
+	TK_ST_BUG,
 	TK_ST_END,
 	TK_ST_ERROR,
-	TK_ST_BUG,
+	TK_ST_IMPORTANT,
+	TK_ST_NEWLINE,
+	TK_ST_OTHER,
+	TK_ST_START,
+	TK_ST_WHITESPACE,
+	TK_ST_WORD,
 }	t_tk_state;
 
 typedef enum e_tk_symbol_type
 {
-	TK_SY_WHITESPACE,
-	TK_SY_LETTER,
 	TK_SY_EOL,
+	TK_SY_ALPHANUM,
+	TK_SY_IMPORTANT,
+	TK_SY_WHITESPACE,
+	TK_SY_NEWLINE,
+	TK_SY_OTHER,
+	TK_SY_BUG,
```

Comment:
Let's name it "Error" instead of "Bug"; Bug is for our errors, "Error" is for user misusing kotishell

Reply:
done